README.md

```diff
@@ -27,43 +27,94 @@ If you don't have `conda` available, we recommend installing [Miniconda for Pyth
 Building from source
 --------------------
 
-This plugin uses [CMake](https://cmake.org/) as its build system.  
-Before compiling you must install [LibTorch](https://pytorch.org/cppdocs/installing.html), which is the PyTorch C++ API, by following the instructions at https://pytorch.org.
-You can then follow these steps:
+### Prerequisites
 
-1. Create a directory in which to build the plugin.
+- Minconda https://docs.conda.io/en/latest/miniconda.html#linux-installers
+- CUDA Toolkit https://developer.nvidia.com/cuda-downloads 
+- LibTorch which is the PyTorch C++ API, by following the instructions at https://pytorch.org/cppdocs/installing.html and https://pytorch.org. 
+    - to get a version for CUDA 1.16 used in our build example we use the commands:
```

Comment:
Please, don't use stuff from pytorch.org. Their binaries are incompatible with `conda-forge`.

Reply:
We probably want to reconsider the build process and possibly make corresponding changes to the CMake script.  If you set `TORCH_DIR` (*not* `PYTORCH_DIR`) to the directory containing `TorchConfig.cmake`, it should be able to automatically find everything it needs.